test/test_genericprocess_tc_cfg_unix.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I'd prefer if these json files were located under test/tests.

Reply:
Changed in latest fixup (to be squashed).